build.sbt

```diff
@@ -1,55 +1,91 @@
+    "-Ywarn-infer-any", // Warn when a type argument is inferred to be `Any`.
+    "-Ywarn-nullary-override", // Warn when non-nullary `def f()' overrides nullary `def f'.
+    "-Ywarn-nullary-unit", // Warn when nullary methods return Unit.
+    "-Ywarn-numeric-widen", // Warn when numerics are widened.
+    "-Ywarn-unused", // Warn if unused.
+    "-Ywarn-value-discard" // Warn when non-Unit expression results are unused.
+  ),
+  scalacOptions in (Compile, console) ~= (_.filterNot(
+    Set(
+      "-Ywarn-unused:imports",
+      "-Xfatal-warnings"
+    ))),
+  scalacOptions in (Test, compile) ~= (_.filterNot(
+    Set(
+      "-Ywarn-unused:imports",
+      "-Xfatal-warnings"
+    ))),
+  resolvers ++= Seq[Resolver](
+    Resolver.sonatypeRepo("releases")
+  )
 )
 
-wartremoverErrors ++= Warts.unsafe
+scalafmtOnCompile := true // TODO: add formatter config
 
-scalacOptions ++= Seq(
-  "-deprecation", // Emit warning and location for usages of deprecated APIs.
-  "-encoding",
-  "utf-8", // Specify character encoding used by source files.
-  "-explaintypes", // Explain type errors in more detail.
-  "-feature", // Emit warning and location for usages of features that should be imported explicitly.
-  "-unchecked", // Enable additional warnings where generated code depends on assumptions.
-  "-Xcheckinit", // Wrap field accessors to throw an exception on uninitialized access.
-  "-Xfatal-warnings", // Fail the compilation if there are any warnings.
-  "-Xlint",
-  "-Yno-adapted-args", // Do not adapt an argument list (either by inserting () or creating a tuple) to match the receiver.
-  "-Ypartial-unification", // Enable partial unification in type constructor inference
-  "-Ywarn-dead-code", // Warn when dead code is identified.
-  "-Ywarn-extra-implicit", // Warn when more than one implicit parameter section is defined.
-  "-Ywarn-inaccessible", // Warn about inaccessible types in method signatures.
-  "-Ywarn-infer-any", // Warn when a type argument is inferred to be `Any`.
-  "-Ywarn-nullary-override", // Warn when non-nullary `def f()' overrides nullary `def f'.
-  "-Ywarn-nullary-unit", // Warn when nullary methods return Unit.
-  "-Ywarn-numeric-widen", // Warn when numerics are widened.
-  "-Ywarn-unused", // Warn if unused.
-  "-Ywarn-value-discard" // Warn when non-Unit expression results are unused.
-)
+wartremoverErrors ++= Warts.unsafe
 
-scalacOptions in (Compile, console) ~= (_.filterNot(
-  Set(
-    "-Ywarn-unused:imports",
-    "-Xfatal-warnings"
-  )))
+// Projects definitions
 
-scalacOptions in (Test, compile) ~= (_.filterNot(
-  Set(
-    "-Ywarn-unused:imports",
-    "-Xfatal-warnings"
-  )))
+lazy val core = (project in file("core"))
+  .settings(commonSettings: _*)
+  .settings(
+    inThisBuild(
+      List(
+        version := "1.0-SNAPSHOT"
+      )),
+    name := "query-core",
+    libraryDependencies ++= Seq(
+      jdbc,
+      Dependencies.anorm,
+      Dependencies.cats,
+      Dependencies.play,
```

Comment:
Does core really need to depend on play ? It think it should just depend on anorm 

And if we really want to use `play.api.database` then just use `play-jdbc`

Reply:
ok I didn't know about `play-jdbc`, I update the dep

Reply:
Or even remove playJdbc with only JDBC in core and play in play sample app